test/XRP/serializer.test.ts

```diff
@@ -266,4 +263,56 @@ describe('serializer', function (): void {
     }
     assert.deepEqual(serialized, expectedJSON)
   })
+
+  it('serializes a payment with a memo', function(): void {
+    // GIVEN a transaction which represents a payment to a destination without a tag, denominated in XRP, with a dank
+    // meme for a memo
+    const transaction = makeTransaction(
+      value,
+      destinationXAddressWithoutTag,
+      fee,
+      lastLedgerSequence,
+      sequence,
+      accountClassicAddress,
+      publicKey,
+    )
+
+    const memo = new Memo()
+    const memoData = new MemoData()
+    memoData.setValue('I forgot to pick up Carl...')
```

Comment:
You could factor these to constants, which would still be funny and then the test is marginally more maintainable at the cost of being more verbose